packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -255,7 +255,7 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 					{DataRow("Container", containerState.id, columnSizing_unstable)}
 					{ContainerStatusRow(statusComponents)}
 					{DataRow("Client ID", containerState.clientId, columnSizing_unstable)}
-					{DataRow("Audience ID", containerState.audienceId, columnSizing_unstable)}
+					{DataRow("User ID", containerState.audienceId, columnSizing_unstable)}
```

Comment:
If the other aspects of this PR end up having long-running discussions, we could definitely pull this change out into its own tiny PR - this is definitely a bug fix IMO

Reply:
+

Reply:
Will make this as a separate PR.